config/datatypes_conf.xml.sample

```diff
@@ -160,6 +160,7 @@
     <datatype extension="loom" type="galaxy.datatypes.binary:Loom" description="An HDF5-based Loom File" mimetype="application/octet-stream" display_in_upload="true"/>
     <datatype extension="h5ad" type="galaxy.datatypes.binary:Anndata" description="An HDF5-based anndata File" mimetype="application/octet-stream" display_in_upload="true"/>
     <datatype extension="mz5" type="galaxy.datatypes.binary:H5" subclass="true" mimetype="application/octet-stream" display_in_upload="true"/>
+    <datatype extension="hyphy_json_results" type="galaxy.datatypes.text:Json" mimetype="application/json" subclass="true" display_in_upload="false"/>
```

Comment:
what do you think about this:
```suggestion
    <datatype extension="hyphy_results.json" type="galaxy.datatypes.text:Json" mimetype="application/json" subclass="true" display_in_upload="false"/>
```
When users download the file it will then be called `dataset.hyphy_results.json` instead of `dataset.hyphy_json_results `, which might be a little nicer

Reply:
That sounds good!

Reply:
Can you accept the commit then ?